common/rpc/encryption/certificate.go

```diff
@@ -54,7 +54,7 @@ func GenerateSelfSignedX509CA(commonName string, extUsage []x509.ExtKeyUsage, ke
 			Organization: []string{"TemporalTechnologiesTesting"},
 		},
 		NotBefore:             now.Add(-time.Minute),
-		NotAfter:              now.AddDate(3, 0, 0), // 3 year expiry
+		NotAfter:              now.AddDate(0, 0, 1), // 1 day expiry
```

Comment:
Why is this?

Reply:
Not really required, but I thought there's no need to generate test certs valid for 3 years.